packages/node-sdk/paima-sm/src/types.ts

```diff
@@ -699,12 +700,13 @@ export type GameStateTransitionFunctionRouter<Events extends AppEvents> = (
   blockHeight: number
 ) => GameStateTransitionFunction<Events>;
 
-export type GameStateTransitionFunction<Events extends AppEvents> = (
-  inputData: STFSubmittedData,
-  blockHeader: PreExecutionBlockHeader,
-  randomnessGenerator: any,
-  DBConn: PoolClient
-) => Promise<{
+export type BaseStfInput = {
```

Comment:
Currently unused, but I need to change `GameStateTransitionFunction` to use it (it's a breaking change though)